### ISSUE TITLE
Codecov returns (404) Not Found

### ISSUE THREAD

**Issue by scott-xu** [codecov/codecov-exe#37]
https://ci.appveyor.com/project/Ninject/ninject-extensions-childkernel/build/3.3.0-ci.17+sha.c7d1687

**Comment by AdmiringWorm** [comment]
Thank you for reporting this issue, I'm looking into the cause of it as we speak.

**Comment by AdmiringWorm** [comment]
I think this may be related to what I tried to fix in the current release.

Still looking into how to actually get this fixed though.

**Comment by AdmiringWorm** [comment]
@scott-xu I just made some changes to the library (not pushed to github yet), and uploaded a beta package to a personal myget repository.

Could I ask you test out these changes before I make any release?
I see you're installing it directly with nuget which means you can change it out with the following to test it out:
```
nuget install Codecov -Version 1.0.5-beta0001 -Source "https://www.myget.org/F/wormie-nugets/api/v3/index.json" -OutputDirectory packages -Verbosity quiet
```

**Comment by scott-xu** [comment]
It works. 
https://ci.appveyor.com/project/Ninject/ninject-extensions-childkernel/build/3.3.0-ci.31+sha.a8ca2f8. 
Thank you @AdmiringWorm

**Comment by AdmiringWorm** [comment]
@scott-xu that's great to hear.
I'm just waiting on confirmation that it also fixes #36, then I'll get a new official release up and running on nuget.org.

I'll let you know when that happens.

**Comment by AdmiringWorm** [comment]
Version 1.0.5 is released.
Still being indexed on nuget.org though.

A chocolatey package for that version have also been released, and is going through moderation now.
